builds/delete-serverless-service-build-pack/Jenkinsfile

```diff
@@ -1269,3 +1325,45 @@ def parseJsonMap(jsonString) {
 	lazyMap.putAll(jsonObj)
 	return lazyMap
 }
+
+def isAzure() {
+  return  service_config['platform'] != null && service_config['platform'] == "azure"
+}
+
+def deleteResourceByTag(env) {
+
+  def stackName = "${configLoader.INSTANCE_PREFIX}-${service_config['domain']}-${service_config['service']}-${env}"
+
+  def serviceInfo = [
+    'stackName'         : stackName,
+    "repoCredentialId"  : repo_credential_id,
+    'envId'             : env,
+    'serviceCatalog'    : service_config,
+    'storageAccountName': 'NA'
+  ]
+
+  azureUtil.setAzureVar(serviceInfo)
+	withCredentials([
+			[$class: 'UsernamePasswordMultiBinding', credentialsId: 'AZ_PASSWORD', passwordVariable: 'AZURE_CLIENT_SECRET', usernameVariable: 'UNAME'],
+			[$class: 'UsernamePasswordMultiBinding', credentialsId: 'AZ_CLIENTID', passwordVariable: 'AZURE_CLIENT_ID', usernameVariable: 'UNAME'],
+			[$class: 'UsernamePasswordMultiBinding', credentialsId: 'AZ_TENANTID', passwordVariable: 'AZURE_TENANT_ID', usernameVariable: 'UNAME'],
+			[$class: 'UsernamePasswordMultiBinding', credentialsId: 'AZ_SUBSCRIPTIONID', passwordVariable: 'AZURE_SUBSCRIPTION_ID', usernameVariable: 'UNAME']
+	]) {
+    def repo_name = "jazz_azure-create-service"
```

Comment:
What is this used for (during deletion)?

Reply:
this is to delete azure resources via that nodejs SDK project.  So we retrieve azure principle from jenkins credentials and call our nodejs project to delete resesources